docs/packages/pkg/msgpack.rst

```diff
@@ -18,9 +18,10 @@ msgpack
 -  Added by `Antal Tátrai <https://github.com/tatraian>`__
    (`pr-406 <https://github.com/ruslo/hunter/pull/406>`__)
 - Available since |hunter|
+- Target library name is now changed from "msgpack::msgpack" to "msgpackc-cxx"
```

Comment:
Please reference the package version number since when the target name was changed (in this case it is `v4.1.3`)

Reply:
```suggestion
- Target library name from "msgpack::msgpack" to "msgpackc-cxx" with `v4.1.3`
```